packages/ui-bindings/Agent/agentInterface.tsx

```diff
@@ -47,14 +51,14 @@ export interface Agent {
   type: string
   image: string
   note: string
-  ownedEconomicResources: [number]
-  agentProcesses: [number]
-  agentPlans: [number]
-  agentEconomicEvents: [number]
-  agentCommitments: [number]
-  agentRelationships: [number]
-  agentRoles: [number]
-  agentRecipies: [number]
+  ownedEconomicResources: [EconomicResource]
+  agentProcesses: [Process]
+  agentPlans: [any] // TODO replace with [Plan]
+  agentEconomicEvents: [any] // TODO replace with [EconomicEvent]
+  agentCommitments: [any] // TODO replace with [Commitment]
+  agentRelationships: [AgentRelationship]
+  agentRoles: [AgentRelationshipRole]
+  agentRecipies: [any] // TODO replace with [ResourceClassification]
```

Comment:
The "any" fillers seem like a necessary evil for now, however. Since there are "TODO" messages which are essentially flags to developers, is it worthwhile to generate a report of some kind so we can keep track of "TODO's" similar to how many IDE's do, just in a recorded fashion?